pkg/provisioning/test/pipelines1/pipelines.go

```diff
@@ -0,0 +1,160 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelines1
+
+import (
+	"time"
+
+	"github.com/conduitio/conduit/pkg/connector"
+	"github.com/conduitio/conduit/pkg/pipeline"
+	"github.com/conduitio/conduit/pkg/processor"
+	"github.com/conduitio/conduit/pkg/provisioning/config"
+)
+
+var P1Config = config.Pipeline{
```

Comment:
Looks good! A question and a comment from me:
1. P1Config appears to be unused. So it's either GoLand not being able to find a usage or maybe there's another reason?
2. It would be good to have a comment in the yaml/Go files to remind that the Go files contain the expected Go values.

Reply:
Oh good catch, I didn't notice it, we're actually only using the instances, I can get rid of the config variables. Sure I can add a comment to the yaml files as well 👍